### PR TITLE
Changed Application Structure

### PR DESCRIPTION
**Pull request by ArturCherkanov**

### CODE REVIEW

File: src/components/Navbar/Navbar.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
+.navbar{
```
Comment (nsine) [inline src/components/Navbar/Navbar.css]: Even when moving your styles to separate files - they are still bundled to one CSS file and all styles are global anyway.
Use one of this approach: https://codeburst.io/4-four-ways-to-style-react-components-ac6f323da822
Or if you prefer to write pure css - I see that Create React App has built-in support of local-scoped css using sass modules
```
// Opt-in support for SASS (using .scss or .sass extensions).
          // Chains the sass-loader with the css-loader and the style-loader
          // to immediately apply all styles to the DOM.
          // By default we support SASS Modules with the
          // extensions .module.scss or .module.sass
```

File: src/Pages/Home/components/MainContent/MainContent.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        <div className="container">
+          <div className="main-content">
+            <div className="calendar-container">
+              <Calendar onClickDay={(value) => {this.sortEventsBySelectedDate(value)}} />
+              <button className="addNewEvent-create-button shadow" onClick={this.activateWindow}>Create</button>
+            </div>
+            <EventList handleSearch={this.handleSearch} chosenEventOnCaledar={this.state.eventList} />
+          </div>
+        </div>
+        <AddNewEvent AddNewEventToDB={this.AddNewEventToDB} />
```
[the remark this answers is not in the document]
Comment (ArturCherkanov) [reply src/Pages/Home/components/MainContent/MainContent.js]: It's a modal window

File: src/Pages/Home/actions/currentEventListAction.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { getEvents } from '../../../api/api';
```
[the remark this answers is not in the document]
Comment (nsine) [reply src/Pages/Home/actions/currentEventListAction.js]: You don't need to write `../api/index`, just `../api`. CommonJS and ES6 module system will automatically search for `index.js` file in folder

File: src/Pages/Home/components/EventList/EventList.jsx [hunk 4]
```diff
@@ -111,5 +118,11 @@ const mapStateToProps = state => {
     }
 }
 
+EventList.propTypes = {
```
Comment (nsine) [inline src/Pages/Home/components/EventList/EventList.jsx]: for proptypes - you can use latest syntax features
It's better to make proptypes as static field in class
https://stackoverflow.com/questions/39587496/static-proptypes-vs-react-proptypes/39683633

File: src/Pages/Home/actions/currentEventListAction.jsx [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import { getEvents } from '../../../api/index';
```
Comment (nsine) [inline src/Pages/Home/actions/currentEventListAction.jsx]: You don't need to write ../api/index, just ../api. CommonJS and ES6 module system will automatically search for index.js file in folder

File: src/Pages/Home/components/EventList/EventList.js [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+            searchQuery: '',
+        };
+    }
+
+    componentWillReceiveProps(next) {
+
+        // Reset selected date 
+
+        if (next.chosenEventOnCaledar !== this.props.chosenEventOnCaledar && next.chosenEventOnCaledar != null) {
+            this.props.needUpdate(false);
```
[the remark this answers is not in the document]
Comment (nsine) [reply src/Pages/Home/components/EventList/EventList.js]: still not fixed

File: src/Pages/Home/components/EventList/EventList.jsx [hunk 7]
```diff
@@ -32,16 +35,18 @@ class EventList extends Component {
     }
 
     render() {
+        const getVisibleItems = this.getVisibleItems(this.props.eventList || [], this.state.searchQuery, this.selectedEventToggle(), this.props.needRefresh);
```
Comment (nsine) [inline src/Pages/Home/components/EventList/EventList.jsx]: This is not method. Why is it starting with `get`?

File: src/Pages/Home/components/MainContent/MainContent.css [hunk 8]
```diff
@@ -0,0 +1,46 @@
+/* General Styles */
+
+.main-page{
```
[the remark this answers is not in the document]
Comment (nsine) [reply src/Pages/Home/components/MainContent/MainContent.css]: still not fixed

File: src/Pages/Home/components/EventListItem/EventListItem.js [hunk 9]
```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
+import './EventListItem.css'
+class EventListItem extends Component {
+    render() {
+        return (
+            <li className="event-item" data-value={this.props.event}>
```
[the remark this answers is not in the document]
Comment (nsine) [reply src/Pages/Home/components/EventListItem/EventListItem.js]: still not fixed

File: src/Pages/Home/components/MainContent/MainContent.js [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+      clearInterval(this.state.intervalIsSet);
+      this.setState({ intervalIsSet: null });
+    }
+  }
+
+  sortEventsBySelectedDate = (date) => {
+    let newArrayWithEvents = [],
+      selectedDate = date.toLocaleString().replace(/,.*$/, "");
+    this.props.eventList.filter(event => {
+      let dataParse = moment(event.time.replace(/T.*$/, "").replace(/-/g, "/")).format("MM/DD/YYYY");
```
[the remark this answers is not in the document]
Comment (nsine) [reply src/Pages/Home/components/MainContent/MainContent.js]: take a look

File: src/Pages/Home/components/MainContent/MainContent.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (nsine) [reply src/Pages/Home/components/MainContent/MainContent.js]: i mean this variable is not used anywhere

File: src/Pages/Login/components/Login/Login.jsx [hunk 11]
```diff
@@ -1,8 +1,10 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
+
 import LoginForm from '../../../../components/LoginForm/loginForm';
```
Comment (nsine) [inline src/Pages/Login/components/Login/Login.jsx]: components must be named starting with Capital letter

File: src/components/AddNewEvent/AddNewEvent.js [hunk 12]
```diff
@@ -0,0 +1,95 @@
...
+                        <label className="addNEwEvent-Label">
+                            <span>Room</span>
+                            <select className="default-input">
+                                <option>Test Room 1</option>
+                                <option>Test Room 2</option>
+                                <option>Test Room 3</option>
+                            </select>
+                        </label>
+                        <div className="addNEwEvent-button-container">
+                            <button className="addNewEvent-create-button" onClick={(e) => { e.preventDefault(); this.props.setModalStateFunction("disabled"); this.props.addEvent(this.state.message, this.state.time) }}>CREATE</button>
```
[the remark this answers is not in the document]
Comment (nsine) [reply src/components/AddNewEvent/AddNewEvent.js]: still not fixed

File: src/reducers/eventsList.js [hunk 13]
```diff
@@ -0,0 +1,10 @@
+export default function eventList(state = "null", action) {
```
[the remark this answers is not in the document]
Comment (nsine) [reply src/reducers/eventsList.js]: not fixed

File: src/reducers/eventsList.jsx [hunk 14]
```diff
@@ -1,4 +1,4 @@
-export default function eventList(state = "null", action) {
+export default function eventList(state = "", action) {
```
Comment (nsine) [inline src/reducers/eventsList.jsx]: we spoke about this. Should be object that really represents state

**Pull request merged**